cmd/rpc.go

```diff
@@ -47,11 +48,11 @@ func RPCFlags() *flag.FlagSet {
 
 func InitClient(cmd *cobra.Command, _ []string) error {
 	if authTokenFlag == "" {
-		storePath := ""
-		if !cmd.Flag(nodeStoreFlag).Changed {
-			return errors.New("cant get the access to the auth token: token/node-store flag was not specified")
+		storePath, err := getStorePath(cmd)
+		if err != nil {
+			return err
 		}
-		storePath = cmd.Flag(nodeStoreFlag).Value.String()
+
```

Comment:
One more thing I realized. We now have a SkipAuth config in rpc module, which removes the need for the token. So we may check for that in node's config and skip getting token. Additionally, cli depends on the default rpc address and port, which may be configurable via config. The really finished goal here would be to cover both of these cases. 

* Discover running node
* Get its config with address and auth information
* Optionally get a token
* Do request.